### PR TITLE
(chore) Update Carbon

### PR DESCRIPTION
**Pull request by Vijaykv5**
## Requirements
- [x] This PR has a title that briefly describes the work done including a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) type prefix and a Jira ticket number if applicable. See existing PR titles for inspiration.

#### For changes to apps
- [ ]  My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide) and [**design documentation**](https://zeroheight.com/23a080e38/p/880723-introduction).

#### If applicable
- [ ] My work includes tests or is validated by existing tests.

## Summary

This PR updates the carbon version to the latest, updates carbon styling imports and removes carbon from declarations.

### CODE REVIEW

**Comment by Vijaykv5** [comment]
@NethmiRodrigo May you can check this out?

File: src/tooltip/tooltip.scss [hunk 1]
```diff
@@ -1,21 +1,21 @@
-@use '@carbon/styles/scss/spacing';
+@use '@carbon/layout';
 @use '@carbon/styles/scss/type';
```
Comment (NethmiRodrigo) [inline src/tooltip/tooltip.scss:2]:
```suggestion
@use '@carbon/type';
```

File: src/tutorial/styles.scss [hunk 2]
```diff
@@ -1,23 +1,23 @@
-@use '@carbon/styles/scss/spacing';
+@use '@carbon/layout';
 @use '@carbon/styles/scss/type';
```
Comment (NethmiRodrigo) [inline src/tutorial/styles.scss:2]:
```suggestion
@use '@carbon/type';
```

**Comment by NethmiRodrigo** [comment]
You're right with the carbon upgrade not fixing the close button rendering properly on the modal, I'm not quite sure yet why... Please give me a bit more time to check

**Comment by Vijaykv5** [comment]
@NethmiRodrigo Still issue with the modal persists
Screenshot 2024-08-15 at 10 04 09 PM

**Comment by Vijaykv5** [comment]
> You're right with the carbon upgrade not fixing the close button rendering properly on the modal, I'm not quite sure yet why... Please give me a bit more time to check

Sure...Thanks @NethmiRodrigo

**Comment by Vijaykv5** [comment]
@NethmiRodrigo Have you checked it out?

**Comment by Vijaykv5** [comment]
This has been fixed over #20

**Pull request closed without merging**